api/src/schema/index.js

```diff
@@ -553,6 +553,34 @@ const Schema = i18n => {
       foundationHeaderHeightMetres
       # ${i18n.t`Filter results for dwellings with a matching header height in feet (ft)`}
       foundationHeaderHeightFeet
+      # ${i18n.t`Filter for dwellings with a matching type of foundation floor type (en)`}
```

Comment:
Hey, I've used the word 'specific' everywhere else, so do you think it would be like-for-like to just reword this, or do you think this fits better?

Personally, I think that "matching" sounds a bit more like equality, which is only one possible way of searching.

Reply:
:dart: